examples/platform/efr32/wf200/host_if.cpp

```diff
@@ -488,679 +495,795 @@ static void sl_wfx_client_connected_callback(uint8_t * mac)
  *****************************************************************************/
 static void sl_wfx_ap_client_rejected_callback(uint32_t status, uint8_t * mac)
 {
-    {
-        // TODO
-        EFR32_LOG("Client rejected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2],
-                  mac[3], mac[4], mac[5]);
-    }
+    EFR32_LOG("Client rejected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2], mac[3],
+              mac[4], mac[5]);
+}
 
-    /****************************************************************************
-     * @fn  static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t *mac)
-     * @brief
-     * Callback for AP client disconnect
-     * @param[in] status:
-     * @param[in]  mac:
-     * @return  None
-     *****************************************************************************/
-    static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t * mac)
-    {
-        // TODO
-        EFR32_LOG("Client disconnected, reason: %d, MAC: %02X:%02X:%02X:%02X:%02X:%02X\r\n", (int) status, mac[0], mac[1], mac[2],
-                  mac[3], mac[4], mac[5]);
-    }
+/****************************************************************************
+ * @fn  static void sl_wfx_ap_client_disconnected_callback(uint32_t status, uint8_t *mac)
```

Comment:
why are we repeating the function signature with `@fn` prefixed? this is already readable in the code. These `@fn` comments are redundant and just eat up vertical space.

Reply:
Thanks for input , I have applied these changes to all other functions comments.